src/platform/packages/private/kbn-scout-info/llms/README.md

```diff
@@ -2,10 +2,34 @@
 
 This guide provides prompts to help you migrate existing FTR tests to the [Scout](https://github.com/elastic/kibana/tree/main/src/platform/packages/shared/kbn-scout) framework.
 
-> [!IMPORTANT]
-> These prompts are experimental. Please carefully review all AI-generated code for mistakes before merging.
+> [!WARNING]
+> These prompts are experimental. Please carefully review all AI-generated code for mistakes before merging. We also highly encourage to make sure the tests you're migrating should be functional tests. If they
+
+## ⚠️ Before migrating a test
+
+First, it's **crucial** to determine if your FTR test should be migrated to a Scout functional test at all. Functional UI tests should focus on **user interactions** and **component rendering**, not on validating the underlying data from APIs. Migrating tests that are essentially data validation checks will lead to a slow and brittle test suite.
+
+### Example: testing a table
+
+Let's suppose you have an FTR test that validates a data table.
+
+- **Avoid** testing the specific contents of the cells. A test asserting that the table has exactly 15 rows and the value in row 3, column 2 is 'Complete' is a data validation test, not a UI functional test. Write an API test instead.
```

Comment:
I think we shouldn't disallow checking number of rows if there a pagination and tests asserts the table e2e rendering. The focus here is to avoid parsing the cells and validating its content.

Reply:
I've rephrased this to make it more clear 👍